cli/template/base/tsconfig.json

```diff
@@ -30,7 +30,7 @@
     }
   },
   "include": [
-    ".eslintrc.cjs",
+    "eslint.config.js",
```

Comment:
this should be picked up by the `**/*.js` glob? reason for the explicitness earlier is files starting with `.` are skipped by the glob